.github/workflows/docker.yml

```diff
@@ -8,7 +8,7 @@ permissions:
   contents: read
 
 env:
-  GO_VERSION: 1.18.4
+  GO_VERSION: 1.19.6
```

Comment:
In the commit message you mention version 1.52.2, but I don't see it here. Am I missing something or is there a typo in the message?

Reply:
the commit message was wrong. I just fixed it. good catch.

Reply:
done